crates/kr/Cargo.toml

```diff
@@ -41,6 +41,7 @@ reqwest = { version = "0.11.0", default_features = false, features = ["json", "r
 askama = "0.10.5"
 notify-rust = "4.5.2"
 run_script = "0.8.0"
+colour = "0.6.0"
```

Comment:
I'll mention that https://crates.io/crates/ansi_term would be a preferred terminal coloring library to use over `colour` given its larger adoption, and greater flexibility.

Reply:
done.